pom.xml

```diff
@@ -682,7 +682,11 @@
           <failFast>false</failFast>
           <rules>
             <banDuplicatePomDependencyVersions/>
-            <requireUpperBoundDeps/>
+            <requireUpperBoundDeps>
+              <excludes>
+                <exclude>org.checkerframework:checker-qual</exclude>
+              </excludes>
+            </requireUpperBoundDeps>
```

Comment:
why is this needed? can you add a note?

Reply:
We need a newer release of Jimfs, added a note